### PR TITLE
Adjust styles mobile view embed maps [WIP]

### PR DESCRIPTION
**Pull request by elenatorro**
Related Issue: https://github.com/CartoDB/cartodb/issues/13487

### CODE REVIEW

File: themes/scss/map/_overlays.scss [hunk 1]
```diff
@@ -521,16 +507,27 @@ $maxLegendContainerHeight: 300px;
   .CDB-Legends-canvas {
     display: none !important;
   }
+
+  .CDB-Legends-canvas.CDB-Legends-canvasEmbed {
+    display: block !important;
```
[the remark this answers is not in the document]
Comment (elenatorro) [reply themes/scss/map/_overlays.scss]: I know, Hound, I am trying to avoid this 💪

File: themes/scss/map/_overlays.scss [hunk 2]
```diff
@@ -164,23 +157,17 @@ $buttonColor: #636D72;
...
   color: $buttonColor;
-
-  &:hover {
-    border-top-left-radius: 4px;
-    border-bottom-left-radius: 4px;
-    background: rgba($cBlue, 0.08);
-  }
 }
 .CDB-Search-text {
+  @include transition(opacity, 150ms ease-in);
```
Comment (matallo) [inline themes/scss/map/_overlays.scss]: do we really need this as mixin? I think we support native `transition` rule

File: themes/scss/map/_overlays.scss [hunk 3]
```diff
@@ -316,86 +302,85 @@ $maxLegendContainerHeight: 300px;
...
-
+.Bubble-numbers {
+  position: relative;
+}
 .Bubble-numbersItem {
-  display: inline-block;
-  padding-right: 5px;
-  transform: translate3d(0, -50%, 0);
+  position: absolute;
+  margin-bottom: -11px; /* to review */
```
Comment (matallo) [inline themes/scss/map/_overlays.scss]: to review?

File: themes/scss/map/_overlays.scss [hunk 4]
```diff
@@ -316,86 +302,85 @@ $maxLegendContainerHeight: 300px;
...
   position: absolute;
-  right: 0;
-  width: 100px;
-  height: 100%;
+  bottom: 0;
+  left: 50%;
 }
 
+.Bubble-container {
+  @include display-flex();
```
Comment (matallo) [inline themes/scss/map/_overlays.scss]: again, I think we can get rid of flex mixin and use it directly

File: themes/scss/map/_overlays.scss [hunk 5]
```diff
@@ -316,86 +302,85 @@ $maxLegendContainerHeight: 300px;
 }
 
 .Bubble-list {
-  width: 100%;
+  @include display-flex();
```
Comment (matallo) [inline themes/scss/map/_overlays.scss]: `display: flex;`

File: themes/scss/map/_overlays.scss [hunk 6]
```diff
@@ -316,86 +302,85 @@ $maxLegendContainerHeight: 300px;
...
+  @include justify-content(center);
```
Comment (matallo) [inline themes/scss/map/_overlays.scss]: `justify-content: center;`

File: themes/scss/map/_overlays.scss [hunk 7]
```diff
@@ -316,86 +302,85 @@ $maxLegendContainerHeight: 300px;
...
-  top: 7px;
-  right: 0;
-  width: 100%;
-  height: 100px;
-  margin-top: 10px;
+  right: 50%;
+  bottom: 0;
+  width: 140px;
+  height: 1px;
+  background: rgba(0, 0, 0, 0.08);
```
Comment (matallo) [inline themes/scss/map/_overlays.scss]: `rgba(#000, 0.08);` better

File: themes/scss/map/_overlays.scss [hunk 8]
```diff
@@ -316,86 +302,85 @@ $maxLegendContainerHeight: 300px;
...
+  content: '';
+  visibility: visible;
 }
-
-.Bubble-circle {
+.Bubble-item {
   @include opacity(0.5);
+  @include transform(translateX(-50%));
```
Comment (matallo) [inline themes/scss/map/_overlays.scss]: use `transform` directly

File: themes/scss/map/_overlays.scss [hunk 9]
```diff
@@ -316,86 +302,85 @@ $maxLegendContainerHeight: 300px;
...
-    height: 1px;
-    background: rgba(#000, 0.08);
-    content: '';
-    z-index: -1;
-  }
+  top: 5px;
+  right: 50%;
+  width: 140px;
+  height: 1px;
+  background: rgba(0, 0, 0, 0.08);
```
Comment (matallo) [inline themes/scss/map/_overlays.scss]: `rgba(#000, 0.08)` better

File: themes/scss/map/_overlays.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (matallo) [reply themes/scss/map/_overlays.scss]: can't we add specificity instead of important?

**Comment by elenatorro** [comment]
I'm currently fixing the specs

**Comment by matallo** [comment]
mixins and comments still pending to be removed

**Pull request merged**